docs/building-for-tv.md

```diff
@@ -4,190 +4,8 @@ title: Building For TV Devices
-  - `onBlur` will be executed when the touchable view goes out of focus
-  - `onPress` will be executed when the touchable view is actually selected by pressing the "select" button on the TV remote.
-
-- _TV remote/keyboard input_: A new native class, `ReactAndroidTVRootViewHelper`, sets up key events handlers for TV remote events. When TV remote events occur, this class fires a JS event. This event will be picked up by instances of the `TVEventHandler` JavaScript object. Application code that needs to implement custom handling of TV remote events can create an instance of `TVEventHandler` and listen for these events, as in the following code:
-
-```tsx
-const TVEventHandler = require('TVEventHandler');
-
-class Game2048 extends React.Component {
-  _tvEventHandler: any;
-
-  _enableTVEventHandler() {
-    this._tvEventHandler = new TVEventHandler();
-    this._tvEventHandler.enable(this, function (cmp, evt) {
-      if (evt && evt.eventType === 'right') {
-        cmp.setState({board: cmp.state.board.move(2)});
-      } else if (evt && evt.eventType === 'up') {
-        cmp.setState({board: cmp.state.board.move(1)});
-      } else if (evt && evt.eventType === 'left') {
-        cmp.setState({board: cmp.state.board.move(0)});
-      } else if (evt && evt.eventType === 'down') {
-        cmp.setState({board: cmp.state.board.move(3)});
-      } else if (evt && evt.eventType === 'playPause') {
-        cmp.restartGame();
-      }
-    });
-  }
-
-  _disableTVEventHandler() {
-    if (this._tvEventHandler) {
-      this._tvEventHandler.disable();
-      delete this._tvEventHandler;
-    }
-  }
-
-  componentDidMount() {
-    this._enableTVEventHandler();
-  }
-
-  componentWillUnmount() {
-    this._disableTVEventHandler();
-  }
-}
-```
-
-- _Dev Menu support_: On the emulator, cmd-M will bring up the Dev Menu, similar to Android. To bring it up on a real Android TV device, press the menu button or long press the fast-forward button on the remote. (Please do not shake the Android TV device, that will not work :) )
-
-- _Known issues_:
-
-  - `TextInput` components do not work for now (i.e. they cannot receive focus automatically, see [this comment](https://github.com/facebook/react-native/pull/16500#issuecomment-629285638)).
-    - It is however possible to use a ref to manually trigger `inputRef.current.focus()`.
-    - You can wrap your input inside a `TouchableWithoutFeedback` component and trigger focus in the `onFocus` event of that touchable. This enables opening the keyboard via the arrow keys.
-    - The keyboard might reset its state after each keypress (this might only happen inside the Android TV emulator).
-  - The content of `Modal` components cannot receive focus, see [this issue](https://github.com/facebook/react-native/issues/24448) for details.
-
-</TabItem>
-<TabItem value="tvos">
-
 > **Deprecated.** TV support has moved to the [React Native for TV](https://github.com/react-native-tvos/react-native-tvos) repository.
```

Comment:
```suggestion
> **Deprecated.** TV support has moved to the [React Native for TV](https://github.com/react-native-tvos/react-native-tvos) repository. Please see the README there for information on projects for Apple TV or Android TV.
```

Reply:
I added this note directing developers to the README in the TV repo, which is kept up to date (including a pointer to Expo TV docs).